### DISCUSSION TITLE
Transparent Border in BSPWM

### DISCUSSION THREAD

**Discussion by XNeyMo** [polybar/polybar#3102] [Q&A] [answered]
Hello, I have problems configuring the transparent border in Polybar, I know what to do and I read the documentation, in addition to the problems page, but no solution could help me.

My problem is that the border of my bar is not transparent, it is black, I don't know why it is black. I am using Arch Linux with BSPWM and Polybar.

This is my config:

`
[global/wm]
margin-bottom = 0pt
margin-top = 0pt
include-file = modules.ini
include-file = colors.ini

[bar/example]
width = 90%
height = 24pt

offset-x = 5%

radius = 20pt

background = ${colors.background}
foreground = ${colors.foreground}

border-size = 4pt
border-color = ${colors.transparent}

padding = 2

module-margin = 0

separator =
dim-value = 1.0

font-0 = monospace;2

modules-left = memory cpu
modules-center = xworkspaces
modules-right = pulseaudio date

cursor-click = pointer
cursor-scroll =

enable-ipc = true
wm-restack = bspwm

[Settings]
screenchange-reload = true
pseudo-transparency = true
`

The value transparent is #00000000, at the top of the image you can see that the border of my bar is black and not transparent
![ss](https://github.com/polybar/polybar/assets/113221957/e2d28b50-4f43-43fc-9b54-5b5c811d321f)

**Comment by patrick96** [comment, marked as answer]
Maybe that's a pasting error, but the `[settings]` has to be all lower case, your config seems to have it capitalized

**Reply by XNeyMo** [reply]
Thanks, i'm sorry, i hadn't seen this
